sources/packages/web/src/components/common/modals/ConfirmExtendTime.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <ModalDialogBase
+    title="Confirm Extend Time"
+    dialogType="warning"
+    :showDialog="showDialog"
+    @click="extendTime"
+  >
+    <template v-slot:content>
+      <v-container class="p-component text-dark">
+        <p>
+          You are about to logged off, do you wish to extend your time?
```

Comment:
You are about to be logged off, do you wish to extend your time? (typo, please add "be")